src/layouts/Settings/GeneralSettings.tsx

```diff
@@ -53,6 +68,30 @@ export const GeneralSettings = ({
             {...register("description")}
           />
         </FormControl>
+        <FormControl isDisabled={isError} isRequired isInvalid={errors.url}>
+          <Box mb="0.75rem">
+            <FormLabel mb={0}>Search Engine Optimisation (SEO)</FormLabel>
+            <FormLabel.Description color="text.description">
+              Enter the web domain of your live site, to improve its position on
+              search result pages.
+            </FormLabel.Description>
+          </Box>
+          <InputGroup>
+            <InputLeftAddon>https://</InputLeftAddon>
+            <Input
+              w="100%"
+              id="url"
+              placeholder="www.open.gov.sg"
+              {...register("url", {
+                required: true,
+                pattern: RegExp(DOMAIN_NAME_REGEX),
+              })}
+            />
+          </InputGroup>
```

Comment:
double checking here - the intent is for them to be secure right (hence the `https://` prefix)? however, when we capture this, we **do not** validate that their site is actually `https` - could we just check this by entering a website prefixed with `http://` like `http://info.cern.ch/`?

Reply:
I added the `^` and `$` part in the regex to enforce that the user would not be able to input `https://` at the front. However, we do not have the ability to check or enforce that the destination site has https enabled unfortunately.

Reply:
yeap, we also don't forcefully append the `https://` prefix here - should we do that then? wrt whether **all** sites have `https` by default - i want to say yes but i'm not sure. i'll just wait for @alexanderleegs to shed some light here and if this is so, then it's ok

Reply:
Yeah they should have - government sites should be using `https://` anyway, and if the cert isn't working, that should probably be fixed before SEO is a concern

Reply:
ok so just to double check - sites will always have `https://` and if their cert expires, how will they know on their side? (i think out of scope here but just piggy backing HAHAHA i think the redir server will resolve hosted sites to `https://www` iirc so i'm fine w/ approving this)

Reply:
I'm guessing the uptime robot checks for any SSL errors? I randomly picked a few sites and they all have the following checked:

<img width="260" alt="Screenshot 2022-08-18 at 15 05 21" src="https://user-images.githubusercontent.com/27919917/185327936-1c295e84-5e44-46b0-954c-64bbbdef1f5a.png">